app/models/distribution.rb

```diff
@@ -3,7 +3,10 @@
 
 class Distribution < BaseModel
   belongs_to :distributable, polymorphic: true, touch: true
+  belongs_to :audio_version_template
```

Comment:
I'm trying to think of a case where a distribution might have > 1 audio version .... but i can't.